health.go

```diff
@@ -791,6 +791,8 @@ type MinioInfo struct {
 	Backend      interface{}  `json:"backend,omitempty"`
 	Servers      []ServerInfo `json:"servers,omitempty"`
 	TLS          *TLSInfo     `json:"tls"`
+	IsKubernetes bool         `json:"is_kubernetes"`
+	IsDocker     bool         `json:"is_docker"`
```

Comment:
Can this always be determined? Otherwise "false" may give a false indication (no pun intended), also for backwards compat all will return "false".

Maybe a `*bool` is more appropriate?

Reply:
Planning to use the functions `IsKubernetes()` and `IsDocker()` from https://github.com/minio/minio/blob/master/cmd/update.go#L154 for populating these.

So I think they'll always be determined. @harshavardhana what do you think?

Reply:
@anjalshireesh Fields will not be present from older server versions. In that case they will be "false".

Reply:
oh yes.. will change to `*bool`